e2e/test/scenarios/dashboard/reproductions/13736-dashboard-should-work-if-some-cards-are-broken.cy.spec.js

```diff
@@ -0,0 +1,48 @@
+import {
+  getDashboardCards,
+  restore,
+  updateDashboardCards,
+  visitDashboard,
+} from "e2e/support/helpers";
+
+describe("issue 13736", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+  });
+
+  it("should work even if some cards are broken (metabase#13736)", () => {
+    cy.intercept("POST", "/api/dashboard/*/dashcard/*/card/1/query", {
+      statusCode: 500,
+      body: {
```

Comment:
I got this response when throwing an error on dashboard card query endpoint with
```clojure
(throw (ex-info "some error" {}))
```